fractal-terrain-generation/src/textureLoader.js

```diff
@@ -0,0 +1,49 @@
+import * as THREE from 'three';
+import * as log from 'loglevel';
+
+import msleep from './msleep';
+
+const loader = new THREE.TextureLoader();
+
+const modifiedRemoteEndpoint = process.env.FRACTAL_ENDPOINT.replace('fractal', 'servePage');
+const resEndpoint = `${process.env.FRACTAL_RESOURCE_ENDPOINT || modifiedRemoteEndpoint}/resources`;
+
+/**
+ * Needed as of now because of our rate limiting.
```

Comment:
Is this still required? If this was related to the sandbox config may be worth revisiting?